addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -107,6 +132,17 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
             lang: "odoo",
             onChange: this._doDebouncedAction.bind(this),
         };
+
+        var fieldNameAttachment = _.first(_.find(_.pairs(this.recordData), function (value) {
```

Comment:
`first` -> `find` is useless

Reply:
...find return [x,y] ... first return the first item...